examples/bzlmod/MODULE.bazel

```diff
@@ -37,6 +37,53 @@ python.toolchain(
 # rules based on the python_versions.
 use_repo(python, "python_3_10", "python_3_9", "python_aliases")
 
+# This extension allows a user to create modifications to how rules_python
+# creates different wheel repositories.  Different attributes allow the user
+# to modify the BUILD file, and copy files.
+# See @rules_python//python/extensions:whl_mods.bzl attributes for more information
+# on each of the attributes.
+# You are able to set a hub name, so that you can have different modifications of the same
+# wheel in different pip hubs.
+whl_mods = use_extension("@rules_python//python/extensions:whl_mods.bzl", "whl_modifications")
+
+# Call whl_mods.create for the requests package.
+whl_mods.create(
+    additive_build_content = """\
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
+write_file(
+    name = "generated_file",
+    out = "generated_file.txt",
+    content = ["Hello world from requests"],
+)
+""",
+    data = [":generated_file"],
+    hub_name = "whl_mods_hub",
+    whl_name = "requests",
+)
+
+# Call whl_mods.create for the wheel package.
+whl_mods.create(
+    additive_build_content = """\
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
+write_file(
+    name = "generated_file",
+    out = "generated_file.txt",
+    content = ["Hello world from build content file"],
+)
+""",
+    copy_executables = {
```

Comment:
These copy arguments are so strange. Any idea why we support this? Why do arbitrary files need to be copied into the package?

Reply:
This is old code 🤷🏻‍♂️ - I’m just creating feature parity.